{"name": "Успешное изменение полей у авторизованного пользователя", "status": "passed", "description": "При попытке сменить password у авторизованного пользователя, изменение данных происходит успешно", "start": 1740399587925, "stop": 1740399589800, "uuid": "2e95210a-fcaf-433e-8911-1448e148b5ff", "historyId": "cacfc362741f1629a5c87885fc13c34c", "testCaseId": "cacfc362741f1629a5c87885fc13c34c", "fullName": "tests.test_changing_user_data.TestChangingUserData#test_changing_user_changing_field", "labels": [{"name": "suite", "value": "Изменение данных пользовователя"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown')"}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "TestChangingUserData"}, {"name": "host", "value": "LAPTOP-JTVFLPC5"}, {"name": "thread", "value": "6028-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_changing_user_data"}]}